bin/node/executor/tests/submit_transaction.rs

```diff
@@ -222,7 +222,7 @@ fn submitted_transaction_should_be_valid() {
 		let extrinsic = UncheckedExtrinsic::decode(&mut &*tx0).unwrap();
 		// add balance to the account
 		let author = extrinsic.signature.clone().unwrap().0;
-		let address = Indices::lookup(author).unwrap();
+		let address = sp_runtime::AccountId32::from(author);
```

Comment:
Why do we need to change to `AccountId32::from`?
What are the difference and consequences?

Reply:
All of modification because of support multi signature of chainbridge. The difference is the address type. Old is Address new is AcountId32.